src/main/scala/Core/CSR.scala

```diff
@@ -35,14 +35,16 @@ class CSR(implicit val conf: FlexpretConfiguration) extends Module {
     val epc = Input(UInt(32.W)) // PC of uncommitted instruction
     val cause = Input(UInt(CAUSE_WI.W))
     val evecs = Output(Vec(conf.threads, UInt(32.W)))
+    val mepcs = Output(Vec(conf.threads, UInt(32.W)))
```

Comment:
mepcs = machine exception PC (plural)
The RISC-V specification mentions three levels of exceptions (M, S, U, if I remember correctly). The M is the standard level to implement